copy_test.go

```diff
@@ -135,6 +135,100 @@ func TestCopy_FullCopy(t *testing.T) {
 	}
 }
 
+func TestCopy_ExistedRoot(t *testing.T) {
```

Comment:
Since the implementation overrides `OnCopySkipped`, we may want to verify that the original `OnCopySkipped` is called.